ui/editor/src/ctrl.ts

```diff
@@ -33,6 +33,7 @@ export default class EditorCtrl {
   epSquare: Square | undefined;
   remainingChecks: RemainingChecks | undefined;
   rules: Rules;
+  variant: string;
```

Comment:
it seems that the unique point of this `variant` member is to tell whether we have chess960 when rules=chess.
So maybe it should be a boolean?